core/src/main/java/org/apache/carbondata/core/util/CarbonTimeStatisticsFactory.java

```diff
@@ -17,47 +17,51 @@
 
 package org.apache.carbondata.core.util;
 
-import org.apache.carbondata.core.carbon.querystatistics.DriverQueryStatisticsRecorder;
+import org.apache.carbondata.core.carbon.querystatistics.*;
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
 
 public class CarbonTimeStatisticsFactory {
   private static String LoadStatisticsInstanceType;
-  private static LoadStatistics LoadStatisticsInstance;
```

Comment:
dont modify here，so that load instsnce can just judge once，if you modify here，it would judge many times.
so， pls set it back.